app/views/jobs/index.html.erb

```diff
@@ -10,7 +10,7 @@
       <h2>Job listings</h2>
       <ul>
         <% @jobs.each do |job| %>
-          <li><%= link_to job.title, job_path(job) %></li>
+          <li><%= link_to job.title, job_path(job) %> <%= job.user.first_name %> needs help on [<%= job.due_date %>] </li>
```

Comment:
If we go down this route we shoudl probably add some interpoaltion to make this more readble, e.g.: `<%= link_to "#{job.title}: ", job_path(job) %>`